### DISCUSSION TITLE
List filter does not render properly

### DISCUSSION THREAD

**Discussion by ronssij** [filamentphp/filament#13856] [Help]
### Package

Table builder

### Package Version

v.3.2.97

### How can we help you?

Filter does not render properly when using `->visible(...)` method base on the page's current active tab.

```php
SelectFilter::make('announceable_id')
     ->visible(fn ($livewire) => FilamentPanel::isAdmin() && ($livewire->activeTab !== 'admins'))
    ->getSearchResultsUsing(fn (string $search) => Clinic::filamentSearch($search))
    ->options(Clinic::query()->limit(50)->pluck('name', 'id'))
    ->placeholder('Search Clinics')
    ->label('Clinic')
    ->searchable()
    ->multiple()
    ->preload(),
```

This is the visual action of the bug:
![CleanShot 2024-08-07 at 16 39 20](https://github.com/user-attachments/assets/0e93be1a-18bc-4386-b4f3-9d03003b912c)

**Comment by FavourOladeji** [comment]
bump. I am also having the same issue. I think it is due to how laravel handles dom diffing
